src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -263,9 +289,6 @@
     //executor service used when vcfInitializerThreads > 1
     private ExecutorService inputPreloadExecutorService;
 
-    @Override
-    public boolean requiresIntervals() { return true; }
```

Comment:
Do we still check for intervals when creating a new GenomicsDB?

Reply:
Yes, initializeIntervals throws an exception if intervals aren't specified and we're not doing incremental import